ConverterService/pkg/repository/currencyUpdate.go

```diff
@@ -0,0 +1,37 @@
+package repository
+
+import (
+	"fmt"
+	"github.com/jmoiron/sqlx"
+	"time"
+)
+
+type updateCurrencyDb interface {
+	updateExchangeRates(baseCurrencies string, quoteCurrencies string, exchangeRates float64) error
+	GetAllPairsData() ([]CurrencyPair, error)
+}
+
+type UpdateCurrencyDb struct {
+	db *sqlx.DB
+}
+
+func NewUpdateCurrencyDb(db *sqlx.DB) *UpdateCurrencyDb {
+	return &UpdateCurrencyDb{db: db}
+}
+
+func (u *UpdateCurrencyDb) UpdateExchangeRates(baseCurrencies string, quoteCurrencies string, exchangeRates float64) error {
```

Comment:
для аргументов с одинаковым типом данных можно задать этот самый тип всего один раз:

```suggestion
func (u *UpdateCurrencyDb) UpdateExchangeRates(baseCurrencies, quoteCurrencies string, exchangeRates float64) error {
```